allure-pytest/test/basic_pytest_features/test_fixture_yield.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.fixture
+def fixture_one():
+    yield
+
+
+def test_func(fixture_one):
```

Comment:
Please,
1) Rename test to ```test_yield_fixture```
2) Renname file to yield_fixture_test.py
3) Move test to  allure-pytest/test/fixtures/function_scope
for more usable for per test running